js/src/structured-data-blocks/how-to/components/HowTo.js

```diff
@@ -443,7 +455,7 @@ export default class HowTo extends Component {
 			<div className={ classNames }>
 				{ ( hasDuration && typeof timeString === "string" && timeString.length > 0 ) &&
 					<p className="schema-how-to-total-time">
-						{ __( "Time needed:", "wordpress-seo" ) }
+						{ durationText }
 						&nbsp;
 						{ timeString }.
```

Comment:
Instead, we should consider to wrap timeString in a span element with a CSS class, to allow theme developers to style it (see screenshot I'll post in the extended comment)